objcToxDemo/OCTConversationViewController.m

```diff
@@ -260,25 +260,25 @@ - (void)toggleMuteMic
 
 - (void)pause
 {
-    OCTCall *call = [self.manager.calls.calls callWithChat:self.chat];
-    [self.manager.calls sendCallControl:OCTToxAVCallControlPause toCall:call error:nil];
+    //    OCTCall *call = [self.manager.calls.calls callWithChat:self.chat];
+    //    [self.manager.calls sendCallControl:OCTToxAVCallControlPause toCall:call error:nil];
 }
 
 - (void)resume
 {
-    OCTCall *call = [self.manager.calls.calls callWithChat:self.chat];
-    [self.manager.calls sendCallControl:OCTToxAVCallControlResume toCall:call error:nil];
+    //    OCTCall *call = [self.manager.calls.calls callWithChat:self.chat];
+    //    [self.manager.calls sendCallControl:OCTToxAVCallControlResume toCall:call error:nil];
 }
 
 - (void)muteFriend
 {
-    OCTCall *call = [self.manager.calls.calls callWithChat:self.chat];
-    [self.manager.calls sendCallControl:OCTToxAVCallControlMuteAudio toCall:call error:nil];
+    //    OCTCall *call = [self.manager.calls.calls callWithChat:self.chat];
+    //    [self.manager.calls sendCallControl:OCTToxAVCallControlMuteAudio toCall:call error:nil];
 }
 
 - (void)unmuteFriend
 {
-    OCTCall *call = [self.manager.calls.calls callWithChat:self.chat];
-    [self.manager.calls sendCallControl:OCTToxAVCallControlUnmuteAudio toCall:call error:nil];
+    //    OCTCall *call = [self.manager.calls.calls callWithChat:self.chat];
+    //    [self.manager.calls sendCallControl:OCTToxAVCallControlUnmuteAudio toCall:call error:nil];
```

Comment:
Yea. Was planning to work on the demo portion once I get all the callSubmanager methods and tests completed.